### ISSUE TITLE
Regression: `line_length` reports violations inside /* … */ and /** … */ comments with `ignores_comments: true`

### ISSUE THREAD

**Issue by justSmK** [realm/SwiftLint#6219]
### New Issue Checklist

- [x] I've updated SwiftLint to the latest version.
- [x] I've searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues).

### Bug Description

When `line_length.ignores_comments` is set to `true`, long lines inside block or doc comments (/* … */, /** … */) still trigger `line_length` violations.
This appears after updating to `0.60.0` (worked on `0.59.1`).

Minimal example
```swift
/**
 This documentation comment line is intentionally very long to reproduce the problem when `line_length.ignores_comments` is set to true. It should be ignored, but it triggers a violation. This documentation comment line is intentionally very long to reproduce the problem when `line_length.ignores_comments` is set to true. It should be ignored, but it triggers a violation.
*/
struct Foo {}
```

[Or GHA example](https://github.com/mindbox-cloud/ios-sdk/actions/runs/17375235113/job/49319971909)

This variant does not trigger a violation:
```swift
/// This documentation comment line is intentionally very long to reproduce the problem when `line_length.ignores_comments` is set to true. It should be ignored, but it triggers a violation. This documentation comment line is intentionally very long to reproduce the problem when `line_length.ignores_comments` is set to true. It should be ignored, but it triggers a violation.
struct Foo {}
```

Mention the command or other SwiftLint integration method that caused the issue. Include stack traces or command output.

```bash
swiftlint --config $CONFIG_FILE
```

### Environment

* SwiftLint version 0.60.0
* Xcode version 16.4. Build version 16F6
* Installation method used Homebrew
* Nested configurations: No
* Configuration file:

```yml
# Directory and file filters
included: # paths to include during linting. `--path` is ignored if present.
  - Mindbox
  - MindboxLogger
  - MindboxNotifications
excluded: # paths to ignore during linting. Takes precedence over `included`.

only_rules: # default rules
  - closing_brace
  - closure_parameter_position
  - colon
  - comma
  - comment_spacing
  - compiler_protocol_init
  - computed_accessors_order
  - control_statement
  - cyclomatic_complexity
  - deployment_target
  - discouraged_direct_init
  - duplicate_conditions
  - duplicate_enum_cases
  - duplicate_imports
  - empty_enum_arguments
  - empty_parameters
  - empty_parentheses_with_trailing_closure
  - file_length
  - for_where
  - force_cast
  - force_try
  - function_body_length
  - function_parameter_count
  - generic_type_name
  - implicit_getter
  - inclusive_language
  - invalid_swiftlint_command
  - is_disjoint
  - large_tuple
  - leading_whitespace
  - legacy_cggeometry_functions
  - legacy_constant
  - legacy_constructor
  - legacy_hashing
  - legacy_nsgeometry_functions
  - legacy_random
  - line_length
  - mark
  - multiple_closures_with_trailing_closure
  - no_space_in_method_call
  - non_optional_string_data_conversion
  - notification_center_detachment
  - ns_number_init_as_function_reference
  - nsobject_prefer_isequal
  - opening_brace
  - operator_whitespace
  - optional_data_string_conversion
  - orphaned_doc_comment
  - prefer_type_checking
  - private_unit_test
  - protocol_property_accessors_order
  - reduce_boolean
  - redundant_discardable_let
  - redundant_objc_attribute
  - implicit_optional_initialization
  - redundant_set_access_control
  - redundant_string_enum_value
  - redundant_void_return
  - return_arrow_whitespace
  - self_in_property_initialization
  - shorthand_operator
  - statement_position
  - static_over_final_class
  - superfluous_disable_command
  - syntactic_sugar
  - trailing_comma
  - trailing_newline
  - trailing_semicolon
  - unavailable_condition
  - unneeded_break_in_switch
  - unneeded_override
  - unneeded_synthesized_initializer
  - unused_closure_parameter
  - unused_control_flow_label
  - unused_enumerated
  - unused_optional_binding
  - unused_setter_value
  - vertical_parameter_alignment
  - vertical_whitespace
  - void_function_in_ternary
  - void_return
  - xctfail_message

  # opt_in_rules
  - attributes
  - closure_body_length
  - closure_end_indentation
  - closure_spacing
  - discouraged_object_literal
  - empty_count
  - empty_string
  - fatal_error_message
  - first_where
  - force_unwrapping
  - last_where
  - modifier_order
  - operator_usage_whitespace
  - private_outlet
  - redundant_type_annotation
  - toggle_bool
  - unneeded_parentheses_in_closure_argument
  - vertical_whitespace_closing_braces
  - weak_delegate
  - yoda_condition

# Default Rules Setup

cyclomatic_complexity:
  ignores_case_statements: true

file_length:
  warning: 400
  error: 800
  ignore_comment_only_lines: true

function_body_length:
  warning: 150
  error: 250

large_tuple:
  warning: 3
  error: 4

line_length:
  warning: 220
  error: 300
  ignores_comments: true
  ignores_function_declarations: true

# Opt-in Rules Setup

closure_body_length:
  warning: 30
  error: 50
```

**Comment by kenji21** [comment]
Maybe related, `file_length` rule is also reported while `ignore_comment_only_lines: true` setted in the configuration

Confirm it worked on 0.59.1

**Comment by SimplyDanny** [comment]
> Maybe related, `file_length` rule is also reported while `ignore_comment_only_lines: true` setted in the configuration
> 
> Confirm it worked on 0.59.1

#6231 should improve this. Please provide an example to be sure.
